tools_docker/Debian_Uwsgi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch
+FROM debian:buster
```

Comment:
Is this required ?
I agree to do this, but this also mean that when stretch support will break we will not be able to know it.
I suggest discuss this with @PhilippeAccorsi .

Reply:
Reverted to using stretch.